keywords/TestServeriOS.py

```diff
@@ -122,7 +146,14 @@ def install(self):
             if self.debug_mode:
                 self.app_name = "{}-{}-debug.app".format(self.app, self.version_build)
```

Comment:
we should add for debug_mode too

Reply:
add what?

Reply:
the logic if build is not provided , get the build from latest builds. You have added that for release mode, but not for debug mode

Reply:
Added logic for debug mode.